azext_iot/central/services/_utility.py

```diff
@@ -121,3 +121,11 @@ def get_object(data: dict, model: str, api_version) -> object:
         raise CLIError(
             "{} is not available for api version == {}".format(model, api_version)
         )
+
+
+def to_camel_dict(data: dict) -> dict:
+    keys = list(data.keys())
+    for key in keys:
+        data[to_camel_case(key)] = data.pop(key)
```

Comment:
this assumes that there won't be a mixed case dictionary with conflicting keys, correct?

Also weird that this function modifies the given data dictionary and returns it - maybe have this function just return a new dictionary without modifiying the dictionary from the argument?

Reply:
assumption is correct. dictionaries are not custom entities for us.
Agree on second point. Refactored.